src/Promenade.Web/Views/Podcasts/Podcast.cshtml

```diff
@@ -68,4 +68,19 @@
             <paginate paginateModel="@Model.PaginateModel"></paginate>
         </div>
     </div>
+}
+@if (!string.IsNullOrWhiteSpace(Context.Items[Ocuda.Promenade.Controllers.ItemKey.GoogleAnalyticsTrackingCode] as string))
+{
+    <script>
+        var players = document.getElementsByTagName("audio");
+        for (var index = 0; index < players.length; index++) {
+            var episode = players[index].getAttribute("data-episode");
+            players[index].addEventListener("play", function () {
+                gtag('event', 'Podcast play', {
+                    'event_category': '@Model.Podcast.Title',
```

Comment:
This is another place to check for `'` in a podcast title.

Reply:
Resolved in commit f460e7d